src/sentry/killswitches.py

```diff
@@ -129,6 +147,17 @@ class KillswitchInfo:
             "message_type": "message type to randomly partition",
         },
     ),
+    "relay.drop-transaction-metrics": KillswitchInfo(
+        description="""
+        Tell Relay via project config to stop extracting metrics from transactions.
+        Note that this change will not take effect immediately, it takes time
+        for downstream Relay instances to update their caches.
+        """,
+        fields={
+            "project_id": "project ID for which we want to stop extracting transaction metrics",
+        },
+        on_change=_update_project_configs,
```

Comment:
I tested this manually, not sure if I need another test in `test_killswitches.py`.